spec/requests/posts_request_spec.rb

```diff
@@ -1,17 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
+  subject do
+    User.new(name: 'Esi Ogagaoghene', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Nigeria.',
+             posts_counters: 0)
+  end
+
+  before { subject.save }
```

Comment:
- [ ] Ensure your  Spec file includes tests that ensure the response body includes the correct placeholder text. Example : `expect(response.body).to include("Welcome to user index page")` Kindly add this ✍🏻

Reply:
@BenMukebo Welcome to user index page is just a text/description. I already referenced a text in that view already. I don't think this is an issue though.